.ruby-version

```diff
@@ -0,0 +1 @@
+Library/Homebrew/.ruby-version
```

Comment:
IIRC this was where we put the `.ruby-version` file initially, but this breaks `rbenv` for scripts that use `#!/usr/bin/env ruby`.

Reply:
See #17390.

Reply:
Could consider adding this to `.gitignore` and having some Homebrew script to automatically make this symlink.

Reply:
Consider investigating setting `rubyLsp.customRubyCommand` (requires setting `rubyLsp.rubyVersionManager` to `custom`) to Portable Ruby or a shim that uses ruby.sh. That will likely avoid the need for this.

Multi-root workspaces may also work here, though not sure if that's what we want.